src/test/fuzz/banman.cpp

```diff
@@ -109,8 +115,9 @@ FUZZ_TARGET_INIT(banman, initialize_banman)
             BanMan ban_man_read{banlist_file, /* client_interface */ nullptr, /* default_ban_time */ 0};
             banmap_t banmap_read;
             ban_man_read.GetBanned(banmap_read);
-            // Temporarily disabled to allow the remainder of the fuzz test to run while a fix is being worked on:
-            // assert(banmap == banmap_read);
+#ifdef ENABLE_BANMAN_COMPARE
+            assert(banmap == banmap_read);
```

Comment:
nit: wouldn't it be easier to just:

```suggestion
            (void)(banmap == banmap_read);
```

Reply:
Yes, that is shorter, thanks!